core/src/software/aws/toolkits/core/lambda/LambdaSampleEventProvider.kt

```diff
@@ -39,11 +39,13 @@ class LambdaSampleEventProvider(private val resourceResolver: RemoteResourceReso
     }
 }
 
-class LambdaSampleEvent(val name: String, private val contentProvider: () -> CompletionStage<String>) {
+open class LambdaSampleEvent internal constructor(val name: String, private val contentProvider: () -> CompletionStage<String>) {
     val content: CompletionStage<String> by lazy { contentProvider() }
     override fun toString() = name
 }
 
+class SimpleLambdaSampleEvent(name: String, content: String) : LambdaSampleEvent(name, { CompletableFuture.completedFuture(content) })
```

Comment:
Local instead of Simple?

Reply:
Moved it as an inner class of the dialog since that's the only place it's used right now - can break it out again if needed